### PR TITLE
use masks in DREAM M2 tests. Closes #981.

### LINKED ISSUE #981: Use masked DREAM vcfs in M2 integration tests
The DREAM vcfs as used in the challenge are masked so that large SV intervals are ignored.  We should used masked vcfs in M2 tests because otherwise we underestimate sensitivity.

### PR DESCRIPTION
**Pull request by davidbenjamin**
@takuto just a little tweak to the M2 integration tests.

### CODE REVIEW

File: src/test/resources/org/broadinstitute/hellbender/tools/mutect/dream/vcfs/sample_3.vcf [hunk 1]
```diff
@@ -16,92 +16,10 @@
...
 ##LeftAlignVariants="analysis_type=LeftAlignVariants input_file=[] read_buffer_size=null phone_home=STANDARD gatk_key=null read_filter=[] intervals=null excludeIntervals=null interval_set_rule=UNION interval_merging=ALL reference_sequence=/cluster/home/ewingad/hive/indexes/bwa/broad/Homo_sapiens_assembly19.fasta nonDeterministicRandomSeed=false downsampling_type=BY_SAMPLE downsample_to_fraction=null downsample_to_coverage=1000 baq=OFF baqGapOpenPenalty=40.0 performanceLog=null useOriginalQualities=false BQSR=null quantize_quals=-1 defaultBaseQualities=-1 validation_strictness=SILENT unsafe=null num_threads=1 num_cpu_threads=null num_io_threads=null num_bam_file_handles=null read_group_black_list=null pedigree=[] pedigreeString=[] pedigreeValidationType=STRICT allow_intervals_with_unindexed_bam=false logging_level=INFO log_to_file=null help=false variant=(RodBinding name=variant source=is3.truth.vcf) out=org.broadinstitute.sting.gatk.io.stubs.VCFWriterStub NO_HEADER=org.broadinstitute.sting.gatk.io.stubs.VCFWriterStub sites_only=org.broadinstitute.sting.gatk.io.stubs.VCFWriterStub filter_mismatching_base_and_quals=false"
+##contig=<ID=20,length=63025520,assembly=GRCh37>
```
Comment (takutosato) [inline src/test/resources/org/broadinstitute/hellbender/tools/mutect/dream/vcfs/sample_3.vcf]: Not a big deal but I think this contig header line appears twice

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java [hunk 2]
```diff
@@ -63,11 +72,18 @@ public void testDreamTumorNormal(final File tumorBam, final String tumorSample,
...
-        Assert.assertTrue("Sensitivity " + sensitivity + " was lower than required sensitivity of " + requiredSensitivity, sensitivity > requiredSensitivity);
-        Assert.assertTrue(fdr < 0.5);
+
+        // run Concordance
+        final File concordanceSummary = createTempFile("concordance", ".txt");
+        new Main().instanceMain(makeCommandLineArgs(Arrays.asList("-truth", truthVcf.getAbsolutePath(), "-eval", unfilteredVcf.getAbsolutePath(), "-L", "20", "-XL", mask.getAbsolutePath(), "-summary", concordanceSummary.getAbsolutePath()), "Concordance"));
+
+        final List<ConcordanceSummaryRecord> summaryRecords = new ConcordanceSummaryRecord.Reader(concordanceSummary).toList();
+        summaryRecords.forEach(rec -> {
+            if (rec.getTruePositives() + rec.getFalseNegatives() > 0) {
```
Comment (takutosato) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java]: Do we need this `if (predicate)` check here?

Comment (davidbenjamin) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java]: This is to make sure that the denominator of `getSensitivity()` wouldn't be zero.

File: src/test/resources/org/broadinstitute/hellbender/tools/mutect/dream/vcfs/sample_3.vcf (hunk 1 above)
Comment (davidbenjamin) [inline src/test/resources/org/broadinstitute/hellbender/tools/mutect/dream/vcfs/sample_3.vcf]: fixed

**Pull request merged**